packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -144,7 +144,8 @@ CoreElement createHotRestartButton(Framework framework) {
 }
 
 Future<void> maybeShowDebugWarning(Framework framework) async {
-  if (serviceManager.connectedApp != null &&
+  if (!importMode &&
```

Comment:
lets add a TODO to add an offline serviceManager implementation as well. For example, if someone sent us a debug snapshot we would want to know! I don't want to spend 30 minutes wondering why someone has the performance they do if the performance is actually performance from debug mode.

Reply:
done